cdap-cli-tests/src/test/java/co/cask/cdap/cli/CLIMainTest.java

```diff
@@ -523,6 +523,9 @@ public void testWorkflowToken() throws Exception {
     testCommandOutputContains(
       cli, String.format("get workflow token %s %s at node %s scope user key %s", workflow, runId,
                          FakeWorkflow.FakeAction.ANOTHER_FAKE_NAME, FakeWorkflow.FakeAction.TOKEN_KEY), fakeNodeValue);
+
+    // stop workflow
+    testCommandOutputContains(cli, "stop workflow " + workflow, "400: Program not running");
```

Comment:
Do you think it makes sense to do a stop + restart just before line 493 in this test? Although I'm not sure if that will make the test flaky since there is no way to tell how long the fake workflow actually runs?